### DISCUSSION TITLE
Provider creating duplicate calendar events

### DISCUSSION THREAD

**Discussion by SnakeFighter** [kewisch/gdata-provider#311] [Support] [answered]
I'm seeing an issue with two identical events appearing in the Thunderbird calendar when I create one with this addon.

- Create new event in Thunderbird via this addon.
- New event appears correctly.
- Synchronise calendars
- Second, identical event appears in Thunderbird
- No duplicate event is created in Google Calendar. Original event appears correctly.
- Deleting one of the identical events in Thunderbird results in the removal of both. Event also removed from Google calendar.

Any thoughts? Am I doing something wrong?

**Comment by kewisch** [comment]
Super weird. Still happening? I can only imagine maybe you have a second calendar subscribed that shows the event?

**Comment by SnakeFighter** [comment, marked as answer]
Apologies- I had forgotten about filing this bug. I've sinced moved over to the latest Beta version of Thunderbird, so I'm not using Provider atm. Not seeing an equivalent bug in the latest setup!
